lib/probes/MemoryProbe.js

```diff
@@ -0,0 +1,37 @@
+var Q = require('q'),
+	util = require('util'),
+	events = require('events'),
+	helpers = require('../helpers'),
+	debug = require('debug')('bp:probes:MemoryProbe');
+
+function MemoryProbe(id) {
+	if (id) {
+		this.id = id;
+	}
+	events.EventEmitter.call(this);
+}
+
+util.inherits(MemoryProbe, events.EventEmitter);
+
+MemoryProbe.prototype.id = 'MemoryProbe';
+
+MemoryProbe.prototype.setup = function(cfg) {
+	this.timerHandle = null;
+};
+
+MemoryProbe.prototype.start = function(browser) {
+	var me = this;
+	this.timerHandle = setInterval(function() {
+		browser.eval('window.performance.memory').then(function(res) {
```

Comment:
Calling eval with a time from webdriver usually results in an extra network call, since the webdriver would have to talk to selenium. This will require the extension to receive the "stringified" message, parse and execute it, increasing the impact of the test on the final data that we get. 

Also note that when you collect memory with metrics like frames_per_sec, you may notice differences since this is additional javascript that is executed, and results in network calls. Could that be the reason you are getting some inconsistent results in your metrics ?